applications/GeoMechanicsApplication/tests/test_submerged_construction_of_excavation.py

```diff
@@ -0,0 +1,205 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.GeoMechanicsApplication.geomechanics_analysis as analysis
+import KratosMultiphysics.GeoMechanicsApplication.context_managers as context_managers
+from KratosMultiphysics.GeoMechanicsApplication.gid_output_file_reader import (
+    GiDOutputFileReader,
+)
+import test_helper
+
+import os
+
+
+class KratosGeoMechanicsSubmergedConstructionOfExcavation(KratosUnittest.TestCase):
+    stage_info = {
+        "initial_stage": {"time": -1.0, "name": "1_Initial_stage"},
+        "null_step": {"time": 0.0, "name": "2_Null_step"},
+        "wall_installation": {"time": 1.0, "name": "3_Wall_installation"},
+        "first_excavation": {"time": 2.0, "name": "4_First_excavation"},
+        "strut_installation": {"time": 3.0, "name": "5_Strut_installation"},
+        "second_excavation": {"time": 4.0, "name": "6_Second_excavation"},
+        "third_excavation": {"time": 5.0, "name": "7_Third_excavation"},
+    }
+
+    BOTTOM_NODE_IDS = [1, 3, 10, 25, 44, 68, 96, 132, 175, 218, 268, 318, 378, 434, 499, 573, 649, 732, 819, 909, 999,
+                       1102, 1206, 1315, 1434, 1560, 1695, 1828, 1966, 2111, 2259, 2406, 2558, 2725, 2896, 3059, 3239,
+                       3422, 3621, 3818, 4023, 4233, 4455, 4739, 5151, 5563, 5959, 6281, 6607, 6924, 7237, 7541, 7831]
+
+    def setUp(self):
+        super().setUp()
+
+        self.unsaturated_unit_weight_of_clay = 16.0e3  # N/m3
+        self.saturated_unit_weight_of_clay = 18.0e3  # N/m3
+        self.saturated_unit_weight_of_sand = 20.0e3  # N/m3
+        self.unit_weight_of_water = 10.0e3  # N/m3
+        self.model_width = 65.0  # m
+        self.excavation_width = 15.0  # m
+        self.unsaturated_clay_layer_thickness = 2.0  # m
+        self.saturated_clay_layer_thickness = 18.0  # m
+        self.saturated_sand_layer_thickness = 30.0  # m
+        self.excavated_middle_clay_layer_thickness = 8.0  # m
+        self.excavated_lower_clay_layer_thickness = 10.0  # m
+        self.weight_of_diaphragm_wall = 10.0e3  # N / m
```

Comment:
Nitpicking (consistency):
```suggestion
        self.weight_of_diaphragm_wall = 10.0e3  # N/m
```

Reply:
removed spaces